v3/impl.h

```diff
@@ -210,7 +213,11 @@ struct wide_integer<Bits, Signed>::_impl {
     }
 
     constexpr static void wide_integer_from_bultin(wide_integer<Bits, Signed>& self, double rhs) noexcept {
-        if ((rhs > 0 && rhs < std::numeric_limits<uint64_t>::max()) ||
+        constexpr uint64_t max_uint = std::numeric_limits<uint64_t>::max();
+        constexpr int64_t max_int = std::numeric_limits<int64_t>::max();
+        constexpr size_t max_sizet = std::numeric_limits<size_t>::max();
```

Comment:
AFAIK `size_t` is used here as alias for `uint64_t`.
`size_t` looks like harmful type here: it should be replaced with `uint64_t` if it is possible. Different definitions of `size_t` and `unit64_t` could lead to some problems.

Reply:
I personally don't see any pros of using size_t over uint64_t here, let's ask @apolukhin